wenab-app/config/BaseApi.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
what are these comments for?

Reply:
They are to disable warnings from eslint since we know that the warnings are nothing to worry about in this file.